vividus/src/integrationTest/java/org/vividus/configuration/Bean.java

```diff
@@ -14,29 +14,32 @@
  * limitations under the License.
  */
 
-package org.vividus.spring;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+package org.vividus.configuration;
 
+import java.lang.System.Logger.Level;
 import java.util.Locale;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
 
-import org.junit.jupiter.api.Test;
-
-class StringToLocaleConverterTests
+public class Bean
```

Comment:
why is it called bean?

Reply:
any concerns?

Reply:
These guys have lots of concerns
![Painted_Pony_Bean](https://user-images.githubusercontent.com/30923336/69737069-c5129880-1144-11ea-9b36-12cfbd810a90.JPG)

Reply:
if seriously this name says nothing about mission of this class

Reply:
This is just a bean

Reply:
I can use your name if you have a good one, am I?

Reply:
Name of class should give some idea about why this class is needed https://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
So, what's yotut pick for pojo with fields?

Reply:
at least the name should reflect somehow that this bean holds fields set to values passed through spring **converters**

Reply:
Like any other bean covered for us by spring :)